README.md

```diff
@@ -166,11 +174,13 @@ A batch file comprises three different line types which exist on a one : many :
 | Water body Buffering | 80008 |
 | Pilot Participation Payment | 80009 |
 
-## Running the application
+# Setup
 
 The application is designed to run in containerised environments, using Docker Compose in development and Kubernetes in production.
 
-- A Helm chart is provided for production deployments to Kubernetes.
+- A Helm chart is provided for deployments to Kubernetes.
+
+## Configuration
 
 ### Build container image
```

Comment:
I think this section is longer than it needs to be.

I think only the 3rd para is needed here